app/controllers/dashboard/work_form/base_controller.rb

```diff
@@ -30,6 +30,15 @@ def next_page_path
           raise NotImplementedError, 'You must implement this method in your controller subclass'
         end
 
+        helper_method :cancel_path
+        def cancel_path
+          if @work_version.present? && @work_version.persisted?
+            resource_path(@work_version&.uuid)
```

Comment:
Does this avoid the reloading of the work version because the database created the UUID? Otherwise, I'm not sure why you'd need the `&`.

Reply:
Yeah good point. I guess I didn't want to get into a race condition with the db, but it's probably not necessary